src/main/webapp/queue.js

```diff
@@ -30,6 +31,21 @@ async function validateRoom() {
   }
 }
 
+// Initializes share tab with lounge's link
+function updateShareTab() {
+  document.getElementById('share').innerHTML = '';
```

Comment:
is this line needed?

Reply:
No, I just wrote it to be extra safe. I'll remove it.